modules/rosa-sts-setting-up-environment.adoc

```diff
@@ -0,0 +1,245 @@
++
+[source,terminal]
+----
+$ rosa completion > /etc/bash_completion.d/rosa
+----
++
+==== Add this file to the correct location for your operating system. For example, on a Linux machine, run the following command to enable `rosa` bash completion:
++
+[source,terminal]
+----
+$ source /etc/bash_completion.d/rosa
+----
+. Download and install the OpenShift cloud credential operator tool. 
++
+[NOTE]
+====
+These steps require Go version 1.15 or greater installed on your machine. See link:https://golang.org/dl/[Go Downloads] for more information.
+====
++
+[source,terminal]
+----
+$ git clone https://github.com/openshift/cloud-credential-operator.git
+$ cd cloud-credential-operator/cmd/ccoctl
+$ go build .
+$ mv ccoctl /usr/local/bin/ccoctl
+----
+
+. Log in to your Red Hat account with `rosa`.
++
+.. Enter the following command.
++
+[source,terminal]
+----
+$ rosa login
+----
++
+.. Replace `<my_offline_access_token>` with your token.
++
+.Example output
+[source,terminal]
+----
+To login to your Red Hat account, get an offline access token at https://cloud.redhat.com/openshift/token/rosa
+? Copy the token and paste it here: <my-offline-access-token>
+----
++
+.Example output continued
+[source,terminal]
+----
+I: Logged in as '<rh-rosa-user>' on 'https://api.openshift.com'
+----
+
+. Verify that your AWS account has the necessary quota to deploy an {product-title} cluster.
++
+[source,terminal]
+----
+$ rosa verify quota [--region=<region>]
+----
++
+.Example output
+[source,terminal]
+----
+I: Validating AWS quota...
+I: AWS quota ok
+----
++
+[NOTE]
+====
+Sometimes your AWS quota varies by region. If you receive any errors, try a different region.
+====
++
+If you need to increase your quota, go to your link:https://aws.amazon.com/console/[AWS console], and request a quota increase for the service that failed.
++
+After both the permissions and quota checks pass, proceed to the next step.
++
+. Prepare your AWS account for cluster deployment:
++
+.. Run the following command to verify your Red Hat and AWS credentials are setup correctly.  Check that your AWS Account ID, Default Region and ARN match what you expect. You can safely ignore the rows beginning with OCM for now (OCM stands for OpenShift Cluster Manager).
++
+[source,terminal]
+----
+$ rosa whoami
+----
++
+.Example output
+[source,terminal]
+----
+AWS Account ID:               000000000000
+AWS Default Region:           us-east-1
+AWS ARN:                      arn:aws:iam::000000000000:user/hello
+OCM API:                      https://api.openshift.com
+OCM Account ID:               1DzGIdIhqEWyt8UUXQhSoWaaaaa
+OCM Account Name:             Your Name
+OCM Account Username:         you@domain.com
+OCM Account Email:            you@domain.com
+OCM Organization ID:          1HopHfA2hcmhup5gCr2uH5aaaaa
+OCM Organization Name:        Red Hat
+OCM Organization External ID: 0000000
+----
+
+. Install the OpenShift CLI (`oc`), version 4.7.9 or greater, from the `rosa` CLI.
```

Comment:
s/ from the `rosa` CLI / from the ROSA (`rosa`) CLI

OCP changed how they document their CLI name recently, and we need to follow suit.